courses/free-courses-hi.md

```diff
@@ -272,6 +272,7 @@
 * [Flutter Series 2020](https://www.youtube.com/playlist?list=PLDzeHZWIZsTo3Cs115GXkot28i406511Y) - CodeHelp - by Babbar
 * [Flutter Tutorial For Beginners in Hindi](https://youtube.com/playlist?list=PLMkkZSS5OjPIwDyHHKVex6zr008U1-sWM) - Geeks Rank
 * [Flutter Widgets in Hindi](https://www.youtube.com/playlist?list=PLz7ymP4HzwSH3vAnhDWLkO2TLbwGeigl7) - Ahirlog
+* [Master Flutter in Just 8 Hours | Full Course Hindi](https://www.youtube.com/watch?v=j-LOab_PzzU) - Codepur
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. View [these docs](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md#platforms-and-access-notes) for more information. Once you have updated you PR, you can add “changes made” to the title and reviewer will take another look as soon as they are available. Thanks!